shared_model/utils/string_builder.hpp

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef IROHA_SHARED_MODEL_STRING_BUILDER_HPP
+#define IROHA_SHARED_MODEL_STRING_BUILDER_HPP
+
+#include <string>
+
+namespace shared_model {
+  namespace util {
```

Comment:
In previous code base, we use detail namespace for helpers. Better to make it unify: rename this namespace with `detail` or rename others to `util`.